tests/test_analyze_template.py

```diff
@@ -95,6 +96,8 @@ def _test(
         self.assertEqual(refs.variables, template_refs)
         self.assertEqual(refs.failed_visits, failed_visits)
         self.assertEqual(refs.unloadable_partials, unloadable)
+        if template_filters:
```

Comment:
I'd like to assert that `refs.filters` an empty dictionary in the event that the `template_filters` argument is `None`. Mainly to catch any accidental omissions of `template_filters` from future test cases.

Reply:
Done in